config/initializers/commontator.rb

```diff
@@ -1,187 +1,249 @@
-
-  # Proc called with user as argument
-  # Returns true iif the user is an admin
-  # Admins can delete other users' comments and close threads
-  # Default: lambda { |user| false } (no admins)
-  config.user_admin_proc = lambda { |user| user.admin? }
-  
-  # Proc called with user as argument
-  # Returns true iif the user should receive subscription emails
-  # Default: lambda { |user| true } (always receive subscription emails)
-  config.subscription_email_enable_proc = lambda { |user| true }
-
-
-  # Commontable (acts_as_commontable) Configuration
-
-  # What a comment is called in your application
-  # Default: 'comment'
-  config.comment_name = 'comment'
-
-  # Verb used when creating comments (present)
-  # Default: 'post'
-  config.comment_create_verb_present = 'post'
+  # or to reapply JQuery UI styles after Ajax calls
+  # Objects visible in view templates can be accessed
+  # through the view object (for example, view.flash)
+  # However, the view does not include the main application's helpers
+  # Default: lambda { |view| '$("#error_explanation").remove();' }
+  config.javascript_proc = lambda { |view|
+                                    '$("#error_explanation").remove();' }
 
-  # Verb used when creating comments (past)
-  # Default: 'posted'
-  config.comment_create_verb_past = 'posted'
 
-  # Verb used when editing comments (present)
-  # Default: 'modify'
-  config.comment_edit_verb_present = 'modify'
 
-  # Verb used when editing comments (past)
-  # Default: 'modified'
-  config.comment_edit_verb_past = 'modified'
-
-  # The format of the timestamps used by Commontator
-  # Default: '%b %d %Y at %I:%M %p'
-  config.timestamp_format = '%b %d %Y at %I:%M %p'
-
-  # Whether admins can edit other users' comments
-  # Default: false
-  config.admin_can_edit_comments = false
-
-  # Whether users automatically subscribe to a thread when commenting
-  # Default: false
-  config.auto_subscribe_on_comment = false
-
-  # Whether users can edit their own comments
-  # Default: true
-  config.can_edit_own_comments = false # REVIEW: Temporarily disabled
+  # User (acts_as_commontator) Configuration
 
+  # user_name_proc
+  # Type: Proc
+  # Arguments: a user (acts_as_commontator)
+  # Returns: the user's name (String)
+  # Default: lambda { |user| I18n.t('commontator.anonymous') } (all users are anonymous)
+  config.user_name_proc = lambda { |user| I18n.t('commontator.anonymous') }
+
+  # user_link_proc
+  # Type: Proc
+  # Arguments: a user (acts_as_commontator),
+  #            the app_routes (ActionDispatch::Routing::RoutesProxy)
+  # Returns: a path to the user's `show` page (String)
+  # If anything non-blank is returned, the user's name in comments
+  # comments will become a hyperlink pointing to this path
+  # The main application's routes can be accessed through the app_routes object
+  # Default: lambda { |user, app_routes| '' } (no link)
+  config.user_link_proc = lambda { |user, app_routes| '' }
```

Comment:
Unused block argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Unused block argument - `app_routes`. If it's necessary, use `_` or `_app_routes` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.